lib/clients/clients.go

```diff
@@ -191,7 +196,51 @@ func (r *ClusterClients) newAppsClient(clusterName string) (app.Applications, er
 	return client, nil
 }
 
-func (r *ClusterClients) newKubeClient(clusterName string) (*kubernetes.Clientset, error) {
+func (r *ClusterClients) newKubeClient(operator ops.Operator, user ops.UserInfo, clusterName string) (*kubernetes.Clientset, error) {
+	remoteCluster, err := r.Tunnel.GetSite(clusterName)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	csr, key, err := authority.GenerateCSR(user.ToCSR(), nil)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	signResponse, err := operator.SignTLSKey(ops.TLSSignRequest{
+		AccountID:  defaults.SystemAccountID,
+		SiteDomain: clusterName,
+		CSR:        csr,
+	})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	client, err := kubernetes.NewForConfig(&rest.Config{
+		Host: defaults.KubernetesAPIURL,
+		TLSClientConfig: rest.TLSClientConfig{
+			CAData:   signResponse.CACert,
+			CertData: signResponse.Cert,
+			KeyData:  key,
+		},
+		WrapTransport: func(t http.RoundTripper) http.RoundTripper {
+			transport := t.(*http.Transport)
+			transport.DialContext = func(ctx context.Context, network, address string) (net.Conn, error) {
+				return remoteCluster.Dial(
+					&httplib.Addr{Net: "tcp", Addr: "127.0.0.1:3024"},
```

Comment:
nit: Should this be in defines? I believe it's being used [here](https://github.com/gravitational/gravity/blob/sergei/monitoring_namespace/lib/webapi/forwarder.go#L198) and [here](https://github.com/gravitational/gravity/blob/sergei/monitoring_namespace/lib/webapi/forwarder.go#L316) as a literal as well

Reply:
Yeah, good point. Wanted to make it a default but forgot.